CONTRIBUTING.md

```diff
@@ -23,40 +23,100 @@ Anyone interested in contributing should join the mailing list and other
 
 ### Reviewing Pull Requests
 
-If you are new to the group, reviewing pull requests and commenting on issues
-is a great way to get involved!
+If you are new to the group, reviewing pull requests and commenting on issues is
+a great way to get involved!  We strongly encourage and support all our members
+to participate in anyway they can.
 
-Except for urgent or very small grammar or spelling fixes, we leave pull 
-requests open for at least 24 hours, so that others have the chance to 
+Except for urgent or very small grammar or spelling fixes, we leave pull
```

Comment:
hmm perhaps a separate "merging pull requests section?"

And perhaps the definition of small changes can reference the "simple changes" section below?>